internal/utils/utils.go

```diff
@@ -0,0 +1,7 @@
+package utils
+
+import "github.com/labstack/echo/v4"
+
+func ErrorResponse(c echo.Context, statusCode int, message string) error {
+	return c.JSON(statusCode, map[string]string{"error": message})
+}
```

Comment:
I don't think this helper is don't much.. it's not worth the function call.. just inline this

Reply:
Okay